src/bam2fasta.cpp

```diff
@@ -100,12 +100,10 @@ int main(int argc, char* argv[])
     std::string suffix;
 
     if (options.get("uncompressed")) {
-        fact.reset(
```

Comment:
Can't you rebase interactive to this commit https://github.com/PacificBiosciences/bam2fastx/pull/4/commits/eef30549b8d18c7ce2cd66c05c6d4dcb3c5f5c51 and edit it there, instead of having a new commit?